[Contributing.md] Local install instructions (#32975)
mui/material-ui: CONTRIBUTING.md

Alternatively, you can open the Netlify preview of the documentation, and open any demo in Codesandbox. The documentation automatically configures the dependencies to use the preview packages.

> Not just that, you can even locally package and test your changes. In the following example, we are packaging mui-material.

Reviewer: ```suggestion
You can also package and test your changes locally. 
The following example shows how to package `@mui/material`:
```

```
$packages\mui-material> yarn build
$packages\mui-material> cd build
$packages\mui-material> npm pack
```

> This will create a mui-material-x.x.x.tar.gz file in the build folder. Copy the file and move it to the project you want to test in. Then do,

Reviewer: ```suggestion
This will create a `mui-material-x.x.x.tar.gz` file in the build folder. 
Copy the file and move it to the project you want to test in, then run:
```

Alternatively, you can open the Netlify preview of the documentation, and open any demo in Codesandbox. The documentation automatically configures the dependencies to use the preview packages.

You can also package and test your changes locally.

> The following example shows how to package `@mui/material` (you can package any mui module with the following steps).

Reviewer: ```suggestion
The following example shows how to package `@mui/material`, but you can package any MUI module with this process:
```

```
$> cd packages/mui-material # or path to any other mui package
$packages\mui-material> yarn build
$packages\mui-material> cd ./build
$packages\mui-material> npm pack
```

> Navigate to the build folder of your respective package and find a file with the following format `mui-material-x.x.x.tar.gz`. Copy this file and move it to the project you want to test in, then run:

Reviewer: ```suggestion
Navigate to the build folder of your respective package and locate a file with the format `mui-material-x.x.x.tar.gz`. 
Copy this file and move it to the project you want to test in, then run:
```

```
$test-project> npm i ./path-to-file/mui-material-x.x.x.tar.gz
```

> **NOTE**: If you have already installed this package, your changes will not be reflected when you reinstall it. A quick fix for this would be to temporarily bump the version number in your `package.json` before running `yarn build`.

Reviewer: ```suggestion
:::info
If you have already installed this package, your changes will not be reflected when you reinstall it. 
As a quick fix, you can temporarily bump the version number in your `package.json` before running `yarn build`.
:::
```

Alternatively, you can open the Netlify preview of the documentation, and open any demo in Codesandbox. The documentation automatically configures the dependencies to use the preview packages.

You can also package and test your changes locally.
The following example shows how to package `@mui/material`, but you can package any MUI module with this process:

> ```

Reviewer: ```suggestion
```sh
```

$> cd packages/mui-material # or path to any other mui package
$packages\mui-material> yarn build
$packages\mui-material> cd ./build
$packages\mui-material> npm pack
```

Navigate to the build folder of your respective package and locate a file with the format `mui-material-x.x.x.tar.gz`. 
Copy this file and move it to the project directory you want to test in, then run:

> ```

Reviewer: ```suggestion
```sh
```
